src/components/timetable/timetableContainer.js

```diff
@@ -34,20 +34,38 @@ function groupDepartures(departures) {
 }
 
 function getNotes(isSummerTimetable) {
+    // because apparently there may be many
+    let didForceShowPNote = false;
+
     return function getNotesInner(routeSegment) {
         if (!routeSegment.hasRegularDayDepartures) {
             return [];
         }
+
         return routeSegment.notes.nodes
             // Y = Yleisöaikataulu
-            .filter(({ noteType }) => noteType.includes("Y"))
             // V = Ympäri vuoden
             // K = Vain kesäaikataulu
             // T = Vain talviaikatalu
-            .filter(({ noteType }) => (
-                noteType.includes("V") || noteType.includes(isSummerTimetable ? "K" : "T")
-            ))
-            .map(note => note.noteText);
+            .filter(({ noteType, noteText }) => {
+                // Find the pe or p notes and force them to show.
+                if (!didForceShowPNote && /^pe?\s/.test(noteText)) {
```

Comment:
Check if the note describes a friday-only departure and make sure that one hasn't already been force-shown.